content/realtime/channel-params.textile

```diff
@@ -0,0 +1,158 @@
+---
+title: Channel Parameters
+section: realtime
+index: 32
+jump_to:
+  Help with:
+    - Overview#overview
+    - Using params with v1.2+ Ably libraries#using-params-with-lib-v12
+    - Using params with v1.1 or earlier Ably libraries#using-params-with-lib-v11
+    - Using params with non-Ably transports#using-params-with-other-transports
+    - Examples#examples
+  Parameters:
+    - rewind#rewind
+---
+
+h2(#overview). Overview
+
+Channel parameters are a general mechanism by which a client can express properties of a channel, or of its attachment to a channel. A set of channel parameters is simply a set of key/value pairs, where both keys and values are strings; the keys correspond to specific features that Ably defines.
+
+The methods provided for specifying channel parameters, and the currently available features, are outlined below.
+
+h2(#supported-params). Currently supported channel params
+
+- rewind := Used to request that an attachment start from some number of messages or point in time in the past. See "rewind":#rewind for more information
+- delta := **In beta**. Used to request that data payloads should be sent as deltas to the previous payload. "Contact us":https://www.ably.io/contact for more information and supported values
+
+h2(#using-params-with-lib-v12). Using channel params with v1.2+ Ably client libraries
+
+With a client library that is version 1.2 or higher, you can specify channel params as part of the @params@ property of the "channel options":/realtime/channels#channel-options.
```

Comment:
Terms that compare library versions I think should be "earlier" and "later". In this PR there's a mix of terms used ("older", "higher" etc).